src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+  /* I have an issue with my v3.8.1. After clicking Connect to Pocket, I just got a blank page on my Mac
+  So I couldn't verify whether the texts were adequate or not...
+  And I don't know what is the "fetch" function for...
+  fevaoctwh, Oct 7, 2023
+  */
+
+{
+  "about": {
+    "app_name": "Pocket Sync",
+    "new_firmware": "发现新的Pocket固件 v{version}",
+    "update_available": "软件有更新! {version}"
+  },
+  "app": {
+    "app_name": "Pocket Sync",
+    "connect_button": "连接到Pocket",
+    "not_pocket_tip": "目录无法识别, 请确认SD卡已经由Pocket完成初始化"
+  },
+  "clean_files": {
+    "close_button": "关闭",
+    "delete_files_button": "删除文件",
+    "files_found": "共找到{count, plural, =0 {0个可清理文件} one {# 个可清理文件} other {# 个可清理文件} }",
+    "title": "清理文件"
+  },
+  "core_info": {
+    "author": "作者",
+    "controls": {
+      "back": "返回列表",
+      "install": "安装",
+      "required_files": "必要文件",
+      "uninstall": "卸载",
+      "update": "更新"
+    },
+    "inputs": "按键提示",
+    "modal": {
+      "close": "关闭",
+      "input_core": "核心",
+      "no_settings_found": "未发现设置文件!",
+      "reset_all": "重置所有设置"
+    },
+    "requires_license": {
+      "all": "请注意!运行该核心需要取得许可授权",
+      "jotego": "The JOTEGO betas require a JTBETA file from the Patreon in the sponsors section below. \nThe jtbeta.zip should be placed at the root of your SD card then it'll be found & copied to the right place during the \"Required Files\" process."
+    //Myself couldn't fully understand the sentence above
+    },
+    "not_in_inventory": "核心{coreName}尚未加入清单",
+    "platform": {
+      "edit": "编辑",
+      "wikipedia": "维基百科"
+    },
+    "replaced": "此核心已经由<1>{replacementCore}</1>核心替代",
+    "platforms": "平台",
+    "please_wait": "请稍后，检查文件中...",
+    "release_date": "发布日期",
+    "releases": {
+      "release_history": "版本发布历史",
+      "show_previous": "显示之前的版本发布历史信息"
+    },
+    "required_files": "必要文件",
+    "required_files_count": "{count} / {total} 文件",
+    "settings": "配置",
+    "sponsor": "Sponsor",
+    "supports": "Supports",
+    "supports_items": {
+      "cartridges": "卡带",
+      "dock": "底座", //do you mean the 99.99 USD Analogue Dock? If it is, the translation is correct
+      "dock_dac": "DAC解码器", // are u refering this item? https://www.analogue.co/dac
```

Comment:
Yeah, nothing supports it yet but when it does this'll end up getting shown in the UI as enabled

Reply:
Resolved